deployments/deployments.json

```diff
@@ -93,6 +93,7 @@
   "288": { "SpokePool": { "address": "0xBbc6009fEfFc27ce705322832Cb2068F8C1e0A58", "blockNumber": 619993 } },
   "324": {
     "SpokePool": { "address": "0xE0B015E54d54fc84a6cB9B666099c46adE9335FF", "blockNumber": 10352565 },
+    "SpokePoolVerifier": { "address": "0x4437A1Ee2dFb72954F21A44C3F0F0E2CBA5A5C27", "blockNumber": 5160661 },
```

Comment:
hmm why was this added to zksync?

Reply:
Ah; that was intended for 690. My mistake.

Reverted: https://github.com/across-protocol/contracts/pull/570/commits/7af99bdbb105f481a75e3546579741029d66cfe9